.env.example

```diff
@@ -0,0 +1,66 @@
+# Change this for production
+SECRET_KEY=ChangeMeToARandomString
+
+# Turn this to True for development.
+DEBUG=False
+
+# This is used by the tracker mailer to determine where to redirect
+# registration and password resets to.
+DOMAIN=example.com
+
+# If using ngrok, add the ngrok hostname here to allow it to hit your dev
+# instance.  In production, add your tracker domain.
+ALLOWED_HOSTS=localhost,example.com
+
+# This causes django-paypal to use the PayPal sandbox for all donations. Be
+# sure to set this to False in production.
+PAYPAL_TEST=True
+
+# In development, leave this unset to use the sqlite db. In production, set
+# this to your db url.
+# ex. psql://tracker:SecretPassword@127.0.0.1:5432/donations
+# DATABASE_URL=psql://<user>:<pass>@<addr>:<port>/<dbname>
+
+# Local time zone for this installation. Choices can be found here:
+# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
+# although not all choices may be available on all operating systems.
+# In a Windows environment this must be set to your system time zone.
+TIME_ZONE=America/New_York
+
+# https://docs.djangoproject.com/en/3.0/ref/settings/#static-files
+STATIC_ROOT=/var/www/static
+STATIC_URL=/static/
+
+# If installing any additional Django apps that need to be registered, add them here.
+# ADDITIONAL_APPS=app,another_app
+
+SWEEPSTAKES_URL=https://gamesdonequick.com/sweepstakes
+PRIVACY_POLICY_URL=https://gamesdonequick.com/privacy
+GOOGLE_ANALYTICS=UA-deadbeef-1,example.com
```

Comment:
Can these get quick comments.

URLS can just be like "static url links used by the frontend" or something.